packages/camera/camera_platform_interface/lib/src/types/camera_description.dart

```diff
@@ -41,20 +59,24 @@ class CameraDescription {
   /// is from top to bottom in the sensor's coordinate system.
   final int sensorOrientation;
 
+  /// The type of lens the camera has.
+  final CameraLensType lensType;
+
   @override
   bool operator ==(Object other) =>
       identical(this, other) ||
       other is CameraDescription &&
           runtimeType == other.runtimeType &&
           name == other.name &&
-          lensDirection == other.lensDirection;
+          lensDirection == other.lensDirection &&
+          lensType == other.lensType;
 
   @override
-  int get hashCode => Object.hash(name, lensDirection);
+  int get hashCode => Object.hash(name, lensDirection, lensType);
 
   @override
   String toString() {
     return '${objectRuntimeType(this, 'CameraDescription')}('
-        '$name, $lensDirection, $sensorOrientation)';
+        '$name, $lensDirection, $sensorOrientation, $lensType)';
   }
```

Comment:
This should get a test as well. It looks like currently we don't have a `toString` test specifically for `CameraDescription`, which we should (only the indirect test in the wrong package that I'm about to change to not test this part any more).

Reply:
Added